tests/test_potentials.py

```diff
@@ -263,6 +263,23 @@ def test_summed_potential(harmonic_bond_test_system):
         GradientTest().compare_forces_gpu_vs_reference(coords, [params], box, potential, rtol, precision)
 
 
+@pytest.mark.parametrize("num_potentials", [1, 2, 5])
+def test_summed_potential_supports_arbitrary_number_of_potentials(num_potentials, harmonic_bond_test_system):
+    """Verifies that summed potential can handle various numbers of potentials. Previously bug
+    only allowed 1 or 2 potentials."""
```

Comment:
Just to verify: the bug only affected the reference summed potential, and had not been exposed up until now; correct?

Reply:
Correct

Reply:
nit: any reason not to merge this into the above `test_summed_potential`, strengthening it to test arbitrary numbers of potentials?

Reply:
Clarified in docstring https://github.com/proteneer/timemachine/pull/930/commits/fb60b4973149dbeb044a8a4d4772f89e79c4e85a

Reply:
> nit: any reason not to merge this into the above `test_summed_potential`, strengthening it to test arbitrary numbers of potentials?

My reasoning was that the other test is testing the consistency of the test system, while this is testing that arbitrary numbers of potentials can be combined. Will merge them back together.

Reply:
> Will merge them back together.

I have a slight preference for this, but non-blocking if you have a reason to prefer them separate.